### ISSUE TITLE
Videos UI: Localize the video lengths.

### ISSUE THREAD

**Issue by kwight** [Automattic/wp-calypso#57571]
As a followup to D69418-code, the `duration_seconds` property of videos will need to be localized client-side.

**Comment by blackjackkent** [comment]
@kwight I spent some time looking into this yesterday and I'm not confident that there is actually a concept of localization for a timestamp like this. MomentJS's [internationalization tools](https://momentjs.com/docs/#/i18n/) relate to clock time, relative time ("an hour ago"), and dates and have nothing regarding duration, and when I googled outside their docs on the subject I didn't find any indication that this is an issue that anyone processes.

I *think* this might be a moot issue and the existing solution is serviceable.
